virttest/qemu_vm.py

```diff
@@ -1906,7 +1906,7 @@ def __iothread_conflict_check(params):
                 "x-scalable-mode": params.get("iommu_x_scalable_mode"),
                 "dma-drain": params.get("iommu_dma_drain"),
                 "pt": params.get("iommu_pt"),
-                "aw-bits": params.get("iommu_aw_bits"),
+                "aw-bits": params.get_numeric("iommu_aw_bits", 39),
```

Comment:
If `aw-bits` expect int value, I think need to handle `aw-bits` in https://github.com/avocado-framework/avocado-vt/blob/71b094feb40ff9dff1eadebd4025a6b5086adce9/virttest/qemu_devices/qdevices.py#L1243-L1286

Reply:
Thank you! Updated accordingly.